Userland/Applications/SoundPlayer/SoundPlayerWidgetAdvancedView.cpp

```diff
@@ -53,7 +54,18 @@ SoundPlayerWidgetAdvancedView::SoundPlayerWidgetAdvancedView(GUI::Window& window
     m_volume_icon = Gfx::Bitmap::load_from_file("/res/icons/16x16/audio-volume-medium.png"sv).release_value_but_fixme_should_propagate_errors();
     m_muted_icon = Gfx::Bitmap::load_from_file("/res/icons/16x16/audio-volume-muted.png"sv).release_value_but_fixme_should_propagate_errors();
 
-    m_visualization = m_player_view->add<BarsVisualizationWidget>();
+    auto visualization_widget = Config::read_string("SoundPlayer"sv, "Preferences"sv, "VisualizationWidget"sv, "bars"sv);
+    if (visualization_widget == "samples")
+        m_visualization = m_player_view->add<SampleWidget>();
```

Comment:
The naming is the wrong way around IMO, the string is "visualization", and the widget is, well, "visualization widget" :)
I think for now `visualization_widget` -> `visualization` is fine.

Reply:
(and `VisualizationWidget` -> `Visualization` for the config name)